packages/mobile/src/exchange/CeloExchangeButtons.tsx

```diff
@@ -0,0 +1,95 @@
+// VIEW which contains buy and sell buttons for CELO.
+
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button.v2'
+import { CURRENCY_ENUM } from '@celo/utils'
+import { StackNavigationProp } from '@react-navigation/stack'
+import BigNumber from 'bignumber.js'
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, View } from 'react-native'
+import { useSelector } from 'react-redux'
+import { CeloExchangeEvents } from 'src/analytics/Events'
+import ValoraAnalytics from 'src/analytics/ValoraAnalytics'
+import { getCeloBalance } from 'src/goldToken/selectors'
+import { Namespaces } from 'src/i18n'
+import { Screens } from 'src/navigator/Screens'
+import { StackParamList } from 'src/navigator/types'
+import { getDollarBalance } from 'src/stableToken/selectors'
+
+interface Props {
+  navigation: StackNavigationProp<StackParamList, Screens.ExchangeHomeScreen>
+}
+
+export default function CeloExchangeButtons({ navigation }: Props) {
+  const { t } = useTranslation(Namespaces.exchangeFlow9)
+
+  const dollarBalance = useSelector(getDollarBalance)
+  const goldBalance = useSelector(getCeloBalance)
+
+  const hasDollars = new BigNumber(dollarBalance || 0).isGreaterThan(0)
```

Comment:
Could add a TODO here to replace with the minimum display or exchange transaction amount (which we don't have hard coded anywhere), for example if a user sees 0.00 but has 0.001 available it could be a confusing experience

Reply:
Used `DOLLAR_TRANSACTION_MIN_AMOUNT` and `GOLD_TRANSACTION_MIN_AMOUNT`